### PR TITLE
feat: support dim preview

### PR DESCRIPTION
**Pull request by zsien**
support dim preview

### CODE REVIEW

**Review by dengbo11 — APPROVED** [review]

File: insidermodule.cpp [hunk 1]
```diff
@@ -110,6 +145,24 @@ void InsiderModule::installDisplayManager(const QString packageName)
     });
 }
 
+static bool imConfigIsDim() {
+    auto home = QDir::home();
+    QFile file(home.filePath(".xinputrc"));
+   
+    if (file.exists()) {
```
Comment (justforlxz) [inline insidermodule.cpp:152]: open

**Review by justforlxz — CHANGES_REQUESTED** [review]

File: insidermodule.cpp (hunk 1 above)
Comment (zsien) [reply insidermodule.cpp]: done

**Review by justforlxz — APPROVED** [review]

**Pull request merged**